src/libs/CMakeLists.txt

```diff
@@ -41,3 +41,5 @@ export(TARGETS cxxopts FILE CxxOptsConfig.cmake)
 
 
 add_subdirectory(redox)
+
+add_subdirectory(rapidjson)
```

Comment:
I added `rapidjson` as a submodule similar to how you added redox, where the remote repo that I link to is my personal fork (for now).